gdc_client/download/client.py

```diff
@@ -274,16 +274,16 @@ def download_small_groups(self, smalls):
         errors = []
         groupings_len = len(smalls)
 
-        pbar = get_percentage_pbar(len(smalls))
-
         for i, small_group in enumerate(smalls):
-            pbar.update(i + 1)
 
             if not small_group:
                 log.error("There are no files to download")
                 return [], 0
 
             log.debug("Saving grouping {0}/{1}".format(i + 1, groupings_len))
+
+            pbar = get_percentage_pbar(1)
```

Comment:
I'm not using a file progress bar, because it'll show the speed as `1.7b/sec` (or some other nonsense) due to the `max_value=1`.

Reply:
didn't we used to show that nonsense tho? 😆